src/megaclient.cpp

```diff
@@ -5325,12 +5323,24 @@ void MegaClient::sc_userattr()
                             if (*cacheduav != *ituav)
                             {
                                 u->invalidateattr(type);
-#ifdef ENABLE_CHAT
-                                if (type == ATTR_KEYRING)
+                                switch(type)
                                 {
-                                    resetKeyring();
+                                    case ATTR_KEYRING:
+                                    {
+                                        resetKeyring();
+                                        break;
+                                    }
+                                    case ATTR_AUTHRING:     // fall-through
+                                    case ATTR_AUTHCU255:    // fall-through
+                                    case ATTR_AUTHRSA:
+                                    {
+                                        mAuthRings.erase(type);
+                                        getua(u, type, 0);
+                                        break;
+                                    }
```

Comment:
It would be great if we could get the proper update via actionpacket, rather than re-requesting - I presume that has been looked into?  thanks

Reply:
Yes, it would be great. But it seems it would result in much larger actionpackets in the API's cache, so the API team rejected to include the value, but only the most recent version (you may not receive one AP for each update, but only one AP for the latest change).

Reply:
ok, thanks for the info :-)